app/helpers/status_helper.rb

```diff
@@ -22,7 +22,16 @@ def status_label(key)
   def status_panel(deploy)
     content = h deploy.summary
     if deploy.active?
-      content << content_tag(:ul, content_tag(:li, deploy.summary_for_process))
+      content << content_tag(:br)
+      content << "Started "
+      content << content_tag(
+        :span,
+        deploy.start_time.rfc822,
+        data: { time: datetime_to_js_ms(deploy.start_time) },
+        class: 'mouseover'
+      )
+      job = deploy.respond_to?(:job) ? deploy.job : deploy
```

Comment:
I wonder if there is a more clear way to say that this can be called with either a job or a deploy